src/frontend/src/routes/router.tsx

```diff
@@ -16,14 +29,17 @@ const Router = () => {
   );
   return (
     <Routes>
-      <Route
-        path="/ppq"
-        element={<PPQLandingPage enablePiaIntakeForm={PIAIntakeFlag} />}
-      />
-      <Route path="/ppq-form" element={<PPQFormPage />} />
-      <Route path="/ppq-connect" element={<PPQConnectPage />} />
-      <Route path="/pia-intake" element={<PIAIntakeFormPage />} />
-      <Route path="/pia-result" element={<PIAIntakeResultsPage />} />
+      <Route path="*" element={<NotFound />} />
```

Comment:
Logically it makes sense to have NotFound route right at the end. Can we please move it to the bottom, if that does not affect the behaviour?

Reply:
will do it